assets/css/main.css

```
@@ -170,6 +170,28 @@ body {
   text-decoration: none;
 }
 
+#Explanation{
+  font-size: 12pt; 
+}
+
+#Toggle-Explanation{
+  background: #f1652991;
+  font-size: 12pt;
+  border: none;
+  color: white ;
```

Similar here, can you use the CSS Custom Property with a fallback?

```suggestion
  color: #fff;
  color: var(--theme-color);
```